js/ui/overview.js

```diff
@@ -271,14 +271,14 @@ const Overview = new Lang.Class({
         this._panelGhost = new St.Bin({ child: new Clutter.Clone({ source: Main.panel.actor }),
                                         reactive: false,
                                         opacity: 0 });
-        this._overview.add_actor(this._panelGhost);
 
         // Create controls
         this._controls = new OverviewControls.ControlsManager();
         this.viewSelector = this._controls.viewSelector;
 
         // Add our same-line elements after the search entry
         this._overview.add(this._controls.actor, { y_fill: true, expand: true });
+        this._overview.add(this._panelGhost);
```

Comment:
The problem I saw was that the cloned desktop was being shifted vertically to the top because of this ghost, that was there because upstream the panel is at the top. Moving it to the bottom fixed it